utils/spacewalk-common-channels.ini

```diff
@@ -3155,6 +3222,113 @@ gpgkey_id = 2F86D6A1
+archs    = x86_64, aarch64, ppc64le, s390x
+name     = AlmaLinux 9 High Availability (%(arch)s)
+base_channels = almalinux9-%(arch)s
+repo_url = https://mirrors.almalinux.org/mirrorlist/9/highavailability
+
+[almalinux9-nfv]
+label    = %(base_channel)s-nfv
+archs    = x86_64, aarch64, ppc64le, s390x
+name     = AlmaLinux 9 NFV (%(arch)s)
+base_channels = almalinux9-%(arch)s
+repo_url = https://mirrors.almalinux.org/mirrorlist/9/nfv
+
+[almalinux9-plus]
+label    = %(base_channel)s-plus
+archs    = x86_64, aarch64, ppc64le, s390x
+name     = AlmaLinux 9 Plus (%(arch)s)
+base_channels = almalinux9-%(arch)s
+repo_url = https://mirrors.almalinux.org/mirrorlist/9/plus
+
+[almalinux9-resilientstorage]
+label    = %(base_channel)s-resilientstorage
+archs    = x86_64, aarch64, ppc64le, s390x
+name     = AlmaLinux 9 ResilientStorage (%(arch)s)
+base_channels = almalinux9-%(arch)s
+repo_url = https://mirrors.almalinux.org/mirrorlist/9/resilientstorage
+
+[almalinux9-rt]
+label    = %(base_channel)s-rt
+archs    = x86_64, aarch64, ppc64le, s390x
+name     = AlmaLinux 9 RT (%(arch)s)
+base_channels = almalinux9-%(arch)s
+repo_url = https://mirrors.almalinux.org/mirrorlist/9/rt
+
+[almalinux9-saphana]
+label    = %(base_channel)s-saphana
+archs    = x86_64, aarch64, ppc64le, s390x
+name     = AlmaLinux 9 SAPHANA (%(arch)s)
+base_channels = almalinux9-%(arch)s
+repo_url = https://mirrors.almalinux.org/mirrorlist/9/saphana
+
+[almalinux9-sap]
+label    = %(base_channel)s-sap
+archs    = x86_64, aarch64, ppc64le, s390x
+name     = AlmaLinux 9 SAP (%(arch)s)
+base_channels = almalinux9-%(arch)s
+repo_url = https://mirrors.almalinux.org/mirrorlist/9/sap
+
+[almalinux9-uyuni-client]
+name     = Uyuni Client Tools for %(base_channel_name)s
+archs    = %(_x86_archs)s, aarch64, ppc64le, s390x
+base_channels = almalinux9-%(arch)s
+gpgkey_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/EL9-Uyuni-Client-Tools/EL_9/repodata/repomd.xml.key
+gpgkey_id = %(_uyuni_gpgkey_id)s
+gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
+repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/EL9-Uyuni-Client-Tools/EL_9/
+
+[almalinux9-uyuni-client-devel]
+name     = Uyuni Client Tools for %(base_channel_name)s (Development)
+archs    = %(_x86_archs)s, aarch64, ppc64le, s390x
+base_channels = almalinux9-%(arch)s
+gpgkey_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/EL9-Uyuni-Client-Tools/EL_9/repodata/repomd.xml.key
+gpgkey_id = %(_uyuni_gpgkey_id)s
+gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
+repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/EL9-Uyuni-Client-Tools/EL_9/
+
+[almalinux9-epel9]
+label    = epel9-%(base_channel)s
+name     = EPEL 9 for %(base_channel_name)s
+archs    = x86_64, aarch64, ppc64le, s390x
+base_channels = almalinux9-%(arch)s
+gpgkey_url = http://download.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-9
```

Comment:
I did *not* change it here, as it's this key is not available at the clients by default, saame as the client tools, so I guess you will handle it later.